packages/pluggableWidgets/sidebar-web/src/Sidebar.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.web.sidebar.Sidebar" needsEntityContext="true" offlineCapable="true"
+        pluginWidget="true"
+        xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd">
+    <name>Sidebar</name>
+    <description>Display sidebar content conditionally.</description>
+    <icon/>
+    <properties>
+        <propertyGroup caption="General">
+            <propertyGroup caption="General">
+                <property key="contents" type="widgets" required="false">
+                    <caption>Sidebar contents</caption>
+                    <description/>
+                </property>
+                <property key="toggleMode" type="enumeration" defaultValue="push">
+                    <caption>Toggle mode</caption>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="none">None</enumerationValue>
+                        <enumerationValue key="push">Push content aside</enumerationValue>
```

Comment:
Is the different between "shrink content" and "push content" that pushing will also push the content off the screen and shrinking does not?

Reply:
Hmm, I haven't thought about this. Right now, both shrink the main content because we want to use the sidebar in combination with the layout grid. The difference between the shrink modes and the push content mode is that the shrink mode allows the sidebar to remain visible in collapsed mode by defining a collapsed width. It's good to verify that this is the desired behaviour and whether this is in line with the old sidebar. If not, we might need to reconsider whether it's possible to use the layout grid with the sidebar.

Reply:
Okay clear, will converge with Chris about this